openhcl/underhill_core/src/loader/mod.rs

```diff
@@ -607,6 +607,7 @@ pub fn write_uefi_config(
         // Some settings do not depend on host config
         flags.set_measure_additional_pcrs(true);
         flags.set_tpm_locality_regs_enabled(true);
+        flags.set_mtrrs_initialized_at_load(true);
```

Comment:
seems reasonable to include some of the context from the PR description as an inline comment here, explaining why this flag is being set

Reply:
Will update; this also depends on https://github.com/microsoft/mu_msvm/pull/19 landing first